### ISSUE TITLE
New version parameter

### ISSUE THREAD

**Issue by Tocci89** [machinegun/SALSA#21]
Hi Machinegun,
I have a similar issue as the one reported in https://github.com/machinegun/SALSA/issues/10. That one was related to an older version of SALSA. Is there a way to apply the same changes you suggested also in the new code?
Thanks in advance

**Comment by machinegun** [comment]
No, the new version does not need you to specify any parameter like that. You just need to specify the minimum contig length you want to consider for scaffolding.

**Comment by Tocci89** [comment]
Thanks a lot.
I'd like to know also how the previously settable parameters are now handled by the software.

**Comment by machinegun** [comment]
You can take a look at the preprint which describes the algorithm for the new version: https://www.biorxiv.org/content/early/2018/02/07/261149

**Comment by Tocci89** [comment]
Thank you so much!
